test_output/help/image_list.txt

```diff
@@ -8,7 +8,6 @@ knife bmcs image list (options)
         --chef-zero-port PORT        Port (or port range) to start chef-zero on.  Port ranges like 1000,1010 or 8889-9999 will try all given ports until one works.
     -k, --key KEY                    API Client Key
         --[no-]color                 Use colored output, defaults to enabled
-        --compartment-id COMPARTMENT The OCID of the compartment.
```

Comment:
Thanks for clarifying - yes, I misunderstood the intention in the ticket.  I'll try and come up with an implementation that leaves compartment id in the common options so we don't have to manually add compartment id to all the existing commands (or for all new commands).